src/network/core/config.h

```diff
@@ -24,7 +24,11 @@ static const char * const NETWORK_CONTENT_MIRROR_URL            = "/bananas";
 static const char * const NETWORK_MASTER_SERVER_WELCOME_MESSAGE = "OpenTTDRegister";
 
 static const uint16 NETWORK_MASTER_SERVER_PORT    = 3978;         ///< The default port of the master server (UDP)
+#ifdef __EMSCRIPTEN__
+static const uint16 NETWORK_CONTENT_SERVER_PORT   =  443;         ///< The default port of the WebSocket server.
```

Comment:
I am really unsure about this. It is correct, but the location feels kinda meh ...

It could also be moved to `pre.js`, and as such, out of the OpenTTD's codebase (and into the `os/emscripten` folder).

I would love opinions here!

(background: emscripten uses WebSocket to communicate, so we setup a WebSocket on `https://content.openttd.org`, so this client can connect there, and talk the normal protocol to talk to the content service; but this does require another port, and using the default https is the easiest).

Reply:
Feel like pre.js might be nicer. At least it needs the background comment above in the code

Reply:
Turns out it cannot go in `pre.js`, because Emscripten is a bit weird. There is a pending PR to fix that, so I will roll this out if that ever lands :) For now, added the background story, good call!